doc/dune-files.rst

```diff
@@ -2056,15 +2035,15 @@ contexts by writing a ``dune-workspace`` file.
 You can point Dune to an explicit ``dune-workspace`` file with the
 ``--workspace`` option. For instance, it's good practice to write a
 ``dune-workspace.dev`` in your project with all the OCaml versions your
-projects' support, so developers can test that the code builds with all
-OCaml versions by simply running:
+projects' support, so developers can test that the code builds with all OCaml
```

Comment:
No apostrophe needed here, I don’t think.
…with all the OCaml versions your projects support…